packages/react-components/src/components/DevicesButton.tsx

```diff
@@ -268,6 +270,7 @@ const generateDefaultMenuProps = (
           itemProps: {
             styles: menuItemStyles
           },
+          role: 'menuitem',
```

Comment:
`menuitem` is the default role, per https://developer.microsoft.com/en-us/fluentui#/controls/web/contextualmenu#IContextualMenuProps

Why do we need to set it explicitly?

Reply:
Yes for the narrator to read out what # of the list the item is it needed that tag there. at least that's what I found when testing. do you have a suggestion on another way?

Reply:
Without the explicit declaration of it being a menu item the windows narrator does not read out which number in the menu item it is for some reason. so like "speaker x item 1 of 2" and so forth.